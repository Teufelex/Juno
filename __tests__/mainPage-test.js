import React from 'react';
import renderer from 'react-test-renderer';

import PageMain from '../pages/Page_Main';

test('работа MainPage', () => {

    const component = renderer.create(
        <PageMain />
    );

    let componentTree=component.toJSON();
    expect(componentTree).toMatchSnapshot();

});