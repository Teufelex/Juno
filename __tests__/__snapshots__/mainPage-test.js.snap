// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`работа MainPage 1`] = `
<div
  className="MainPage"
>
  <div
    className="MainPage__titleWrapper"
  >
    <h2
      className="titleWrapper__title"
    >
      Make your incredible travel
    </h2>
  </div>
  <div
    className="AboutPage"
  >
    <div
      className="AboutPage__block"
    >
      <img
        className="AboutPage__img"
        src="/assets/globe.jpg"
      />
      <p>
        Добро пожаловать в Juno! Уже 228 лет подряд, мы - самая передовая туристическая компания Солнечной Системы, которая предоставит вам тур в любой обитаемый (и не очень) космический объект ближнего космоса. Все путешествия производятся на кораблях Extra-Luxe класса, произведенные компанией "Xpase-S" - лидером по производству летательных аппаратов из радиоактивных отходов и тяжелых металлов!
      </p>
    </div>
    <div
      className="AboutPage__block AboutPage__block--reverse"
    >
      <p>
        На наших космолетах присутствуют все необходимые системы жизнеобеспечения - вода, электричество, газ, бесплатное пиво и безлимитный доступ к спортивным каналам (для мужчин), а так же красное вино и бесконечный повтор фильма "Титаник" для прекрасной половины населения галактики. На нижних ярусах космолета находятся плантации с самыми необходимыми для туристов растениями, начиная от красного помидора и заканчивая табаком (не срывайте всё - они же обеспечивают нас кислородом!). Для особенных ценителей установлены шезлонги, позволяющие вернуться в аутентичную атмосферу родной деревни, а для тех самых, кому по карману оценить все преимущества, мы даже выпустим стаю комаров для полного погружения.
      </p>
      <img
        className="AboutPage__img"
        src="/assets/plant.jpg"
      />
    </div>
    <div
      className="AboutPage__block"
    >
      <img
        className="AboutPage__img"
        src="/assets/bedroom.jpg"
      />
      <p>
        Спальные места на наших космолетах так же оборудованы по высшему разряду. Кровати из лучших обшивок разбившихся космолетов, мягкий матрас из шерсти кошки, удобная подушка, набитая перьями марсианского голубя, а так же одеяло из конопляной пряжи для красочных снов - разве не идеально? В каждой каюте вас будет ждать галактическое кабельное с самыми лучшими каналами, для досуга и расслабления вашего мозга перед сном. Для самых обеспеченных ценителей полетов в космосе будут доступны каюты с видом на звезды, планеты и космический мусор. Путешествуйте с нами и помните - лучшего турагенства вам не найти, мы ведь единственные в галактике!
      </p>
    </div>
  </div>
</div>
`;
